superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx

```diff
@@ -261,17 +274,17 @@ const ChartHolder: React.FC<ChartHolderProps> = ({
           onResizeStop={onResizeStop}
           editMode={editMode}
         >
-          <div
+          <FullSizeWrapper
```

Comment:
Another way of writing this logic is to compile the CSS and apply it only if necessary:

```
const fullSizeStyle = css`
  && {
    position: fixed;
    z-index: 3000;
    left: 0;
    top: 0;
  }
`;

<div
   css={isFullSize ? fullSizeStyle : undefined}
>
```